proto/substrait/algebra.proto

```diff
@@ -726,6 +730,47 @@ message NestedLoopJoinRel {
   substrait.extensions.AdvancedExtension advanced_extension = 10;
 }
 
+message DelimGetRel{
+   RelCommon common = 1;
+   repeated  Type  delim_types = 2;
+}
+
+message DelimJoinRel {
+  RelCommon common = 1;
+  Rel left = 2;
+  Rel right = 3;
+
+  Expression expression = 4;
+  Expression post_join_filter = 5;
+
+  JoinType type = 6;
+
+  // The set of columns that will be duplicate eliminated from the LHS and pushed into the RHS
+  repeated Expression duplicate_eliminated_columns = 7;
```

Comment:
This is the equivalent to a set of field references?  Is it permissible to include any other kinds of expression here (such as a literal)?

Reply:
Absolutely, I'll change these to be only fieldrefs.